morebits.js

```diff
@@ -459,7 +459,6 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 			node = document.createDocumentFragment();
 			// fragments can't have any attributes, so just return it straight away
 			return [ node, node ];
-		// Sometimes Twinkle uses fancy searchable "select" elements. This is powered by the third party library "select2". Activate it by creating a Morebits "select" element, then call `$('select[name=sub_group]').select2({});` or similar towards the end of your main code.
```

Comment:
Can we keep this comment please? Seems helpful.

Reply:
Re-added